packages/core/src/common/command.ts

```diff
@@ -371,7 +362,8 @@ export class CommandRegistry implements CommandService {
      */
     getAllHandlers(commandId: string): CommandHandler[] {
         const handlers = this._handlers[commandId];
-        return handlers ? handlers.slice() : [];
+        // We intentionally reverse the array of handlers, so if there are multiple handlers for a command, you can find the more specific, enabled one.
+        return handlers ? handlers.slice().reverse() : [];
```

Comment:
We should add it to CHANGELOG.

Please add also a point to `How to test` whether we have multiple handlers and check that they work.

Reply:
Added. See the PR description.